bff/src/Bff.Blazor/BffBuilderExtensions.cs

```diff
@@ -4,6 +4,8 @@
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Components.Authorization;
 using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
```

Comment:
Looks like this file didn't need to be included in the PR?

Reply:
I removed it